pkg/logger/logger_test.go

```diff
@@ -0,0 +1,130 @@
+package logger
+
+import (
+	"bytes"
+	"errors"
+	"fmt"
+	"os"
+	"os/exec"
+	"sync"
+	"testing"
+
+	"github.com/rs/zerolog"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+var mu sync.Mutex
+
+type loggerTest struct {
+	name         string
+	logLevel     string
+	testFunction func(t *testing.T, log *Logger, buf *bytes.Buffer)
```

Comment:
hey @emeeker2002 typically you'd want to have an expected result in your test struct. and then in each test case, you'd set the expected result.  That way all assets can be handled in the for loop at the end.

Reply:
This would explain why the linter was wanting t.Helper in each function  which was a bit odd to me.